src/components/aboutUs.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export const AboutUs = () => 
+     (
+         <>
```

Comment:
these fragments need removing please! they weren't in when i originally reviewed, they've been added in since. they need taking back out please. in both files! thankyou! then i can approve :)

Reply:
https://reactjs.org/docs/fragments.html <--- more about fragments. (you don't need them here.)